docs/guides/airnode/calling-an-airnode/index.md

```diff
@@ -47,19 +47,25 @@ Remix and try deploying your own Requester Contract.
 [Open in Remix<ExternalLinkImage/>](https://remix.ethereum.org/#url=https://github.com/vanshwassan/RemixContracts/blob/master/contracts/Requester.sol&optimize=false&runs=200&evmVersion=null&version=soljson-v0.8.9+commit.e5eed63a.js)
 
 ```solidity
-// SPDX-License-Identifier: MIT
+//SPDX-License-Identifier: MIT
 pragma solidity 0.8.9;
 // [!code focus:2]
 import "@api3/airnode-protocol/contracts/rrp/requesters/RrpRequesterV0.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
 
-// A Requester that will return the requested data by calling the specified airnode.
-
-contract Requester is RrpRequesterV0 {
+// A Requester that will return the requested data by calling the specified Airnode.
+contract Requester is RrpRequesterV0, Ownable {
     mapping(bytes32 => bool) public incomingFulfillments;
     mapping(bytes32 => int256) public fulfilledData;
 
-    // Make sure you specify the right _rrpAddress for your chain while deploying the contract.
+// Make sure you specify the right _rrpAddress for your chain while deploying the contract.
```

Comment:
Why remove the indentation here and elsewhere? It looks a bit strange in the contract code: https://vitepress-docs--pr493-requester-with-withd-ggoohfx2.web.app/guides/airnode/calling-an-airnode/#_1-coding-the-requester-contract

Reply:
Fixed 👍🏻